configs/platforms/ubuntu-14.04-amd64.rb

```diff
@@ -1,11 +1,10 @@
-peversion = IO.read('peversion').strip
 platform "ubuntu-14.04-amd64" do |plat|
   plat.servicedir "/etc/init.d"
   plat.defaultdir "/etc/default"
   plat.servicetype "sysv"
   plat.codename "trusty"
 
-  plat.add_build_repository "http://enterprise.delivery.puppetlabs.net/#{peversion}/repos/#{plat.get_name}/#{plat.get_name}.repo"
+  plat.add_build_repository "http://enterprise.delivery.puppetlabs.net/2019.0/repos/#{plat.get_name}/#{plat.get_name}.repo"
```

Comment:
We dropped ubuntu-14.04 support starting in 2018.1, so you can probably just remove this whole line. Looking at other platform configs in this repo, only PE master platforms need this.

Reply:
Thanks, will update